README.md

```diff
@@ -45,6 +45,12 @@ Each **`Input`**, **`Parser`**, **`Filter`**, **`Output`** represents a Fluent B
 
 ![Fluent Bit workflow](docs/images/fluent-bit-operator-workflow.svg)
 
+A reload interface is added to fluent bit for support dynamic configuration so that any changes on the configuration will be detected and reloaded by the agent instead of a restart.
+
+As shown in the diagram below, the config file watcher implements the reloading functionality and is responsible for the life cycle of fluent-bit process.
```

Comment:
`As shown in the diagram below, the config file watcher implements the reloading functionality and is responsible for the life cycle of fluent-bit process.` can be removed